pkg/power/components/source/rapl_msr_util.go

```diff
@@ -47,57 +43,36 @@ const (
 )
 
 var (
+	cpu       *ghw.CPUInfo
 	fds       []int
 	byteOrder binary.ByteOrder
 
-	// package - core
-	packageMap  map[int]int
-	packageList []int
+	numPackages, numCores int
```

Comment:
I am not finding where you set the values in this variables. Shouldn't it be in the `init()` ?

Reply:
Sorry, my bad, let me rebase a single diff.